src/interactive/AccordionPanelGroup.jsx

```diff
@@ -5,12 +5,25 @@ import cx from 'classnames';
 export const ACCORDIONPANELGROUP_CLASS = 'accordionPanelGroup';
 
 /**
- * Determines if the passed in value is a primitive type
- * @param {any} value
- * @return {boolean} True if value is a primitive type
+ * @description takes AccordionPanelGroup and AccordionPanel data and returns an array of new panel `isOpen` states
+ * @param {Boolean} isMultiselect
+ * @param {Array} statesList
+ * @param {Object} panelData
```

Comment:
what do you think of naming this `clickedPanelData` for clarity?

Reply:
Into it